src/components/table/table-theme.scss

```diff
@@ -0,0 +1,30 @@
+.column {
+  height: 100%;
+  display: flex;
+  align-items: flex-end;
+
+  > p {
+    padding: 20px;
+  }
+}
+
+.row {
+  padding-top: 0 !important; // Due to the :global in styles
+  padding-bottom: 0 !important; // Due to the :global in styles
+}
+
+.evenRow {
+  .column {
+    &:nth-child(2) {
+      background-color: #e7edf8;
+    }
+  }
+}
+
+.oddRow {
+  .column {
+    &:nth-child(2) {
+      background-color: #d4dff4;
+    }
+  }
+}
```

Comment:
In the beginning I thought that we can do this simpler, by writing CSS with `&:nth-child(odd)` and `&:nth-child(even)` selectors, so I tried this:
![image](https://user-images.githubusercontent.com/15097138/69866453-fd78ba80-129b-11ea-9e62-392166f618cf.png)

and after that this happened:
![hd9de-36jy8](https://user-images.githubusercontent.com/15097138/69866483-1aad8900-129c-11ea-876d-47dcf3d818cd.gif)


Cells color changes after scrolling - it's because this table uses lazy loading and renders rows dynamically, so rows that were even at the beginning, after scroll can be odd. So this is a good solution to add classes `.even` and `.odd` explicitly :+1: . I just wanted to share it because it was quite interesting to me :D

Great job with this PR @Bluesmile82, you can merge it :wink: